### ISSUE TITLE
Detect workdir if not specified in the manifest

### ISSUE THREAD

**Issue by rberrelleza** [okteto/remote-kubernetes#130]
**Is your feature request related to a problem? Please describe.**
If the `workdir` is not specified in the manifest, we should try to detect it instead of always going to `/usr/src/app`, since it might not be available in the container.

**Additional context**
we could get it from the SSH check we already do.

**Comment by rberrelleza** [comment]
Another option, suggested by @pchico83 

```
if sync is specified, it takes preference. if there is more than one, could we open more than one folder?
if sync is not specified, workdir is mandatory and it is what should be opened.
```

**Comment by rberrelleza** [comment]
this will be fixed on okteto cli 3.0
